src/index.html

```diff
@@ -6,17 +6,49 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
     />
+    <link rel="stylesheet" href="./style.css" />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="Moyo logo" class="img" />
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="list_item">
+            <a href="#" class="nav_link is-active">Apple</a>
+          </li>
+          <li class="list_item">
+            <a href="#" class="nav_link">Samsung</a>
+          </li>
+          <li class="list_item">
+            <a href="#" class="nav_link">Smartphones</a>
+          </li>
+          <li class="list_item">
+            <a href="#" data-qa="hover" class="nav_link">Laptops & Computers</a>
+          </li>
+          <li class="list_item">
+            <a href="#" class="nav_link">Gadgets</a>
+          </li>
+          <li class="list_item">
+            <a href="#" class="nav_link">Tablets</a>
+          </li>
+          <li class="list_item">
+            <a href="#" class="nav_link">Photo</a>
+          </li>
```

Comment:
Add empty lines between multiline sibling blocks of HTML

![image](https://github.com/mate-academy/layout_moyo-header/assets/54860641/ab5ca66b-3cb0-4cfa-be9a-b13059dbf433)